Mathlib/Analysis/Complex/UpperHalfPlane/Basic.lean

```diff
@@ -8,6 +8,7 @@ import Mathlib.LinearAlgebra.Matrix.SpecialLinearGroup
 import Mathlib.Analysis.Complex.Basic
 import Mathlib.GroupTheory.GroupAction.Defs
 import Mathlib.LinearAlgebra.Matrix.GeneralLinearGroup
+import Mathlib.Tactic.AdaptationNote
```

Comment:
General question: should AdaptationNote be added to Tactic.Common? (I'm fine with not. The current situation added a little bit of fraction to the conversion, but that was a one-time event.)

Reply:
Let's leave it out for now.